code/modules/mob/living/simple_animal/slaughter/slaughter.dm

```diff
@@ -0,0 +1,184 @@
+//////////////////The Monster
+
+/mob/living/simple_animal/slaughter
+	name = "Slaughter Demon"
+	real_name = "Slaughter Demon"
+	desc = "You should run."
+	speak_emote = list("gurgles")
+	emote_hear = list("wails","screeches")
+	response_help  = "thinks better of touching"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "horror"
+	speed = 0
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	attack_sound = 'sound/hallucinations/growl1.ogg'
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	faction = list("slaughter")
+	attacktext = "wildly tears into"
+	maxHealth = 250
+	health = 250
+	environment_smash = 1
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	var/devoured = 0
+	var/phased = FALSE
+	var/holder = null
+	var/eating = FALSE
+	var/mob/living/kidnapped = null
+	var/playstyle_string = "<B>You are the Slaughter Demon, a terible creature from another existence. You have a single desire: To kill.  \
+						You may Ctrl+Click on blood pools to travel through them, appearing and dissaapearing from the station at will. \
+						Pulling a dead or critical mob while you enter a pool will pull them in with you, allowing you to feast. </B>"
+
+
+/mob/living/simple_animal/slaughter/death()
+	..(1)
+	new /obj/effect/decal/cleanable/blood (src.loc)
+	new /obj/item/weapon/demonheart (src.loc)
+	visible_message("<span class='danger'>The [src] screams in anger as its form collapes into a pool of viscera.</span>")
+	ghostize()
+	qdel(src)
+	return
+
+
+
+////////////////////The Powers
+
+/mob/living/simple_animal/slaughter/proc/phaseout(var/obj/effect/decal/cleanable/B)
+	var/turf/mobloc = get_turf(src.loc)
+	var/turf/bloodloc = get_turf(B.loc)
+	if(Adjacent(bloodloc))
+		src.notransform = TRUE
+		spawn(0)
+			src.visible_message("The [src] sinks into the pool of blood, and out of existence.")
```

Comment:
"and out of existence" sounds really awkward here, it'd work fine with just the first bit.